test/ParallelIOTest.cpp

```diff
@@ -9,8 +9,18 @@ using namespace openPMD;
 #include <catch/catch.hpp>
 #if openPMD_HAVE_MPI
 #   include <mpi.h>
+#endif
```

Comment:
Except for catch, there's no need for _any_ headers without MPI at the end of the day.

Reply:
;-)

The all-serial use case will be the windows use case xD

Reply:
Since the tests have no headers and use these stdlib containers and algos directly, it's save to include them. Otherwise they are only pulled indirectly and this might change on unknown implementation details.

Or am I missing something?

Reply:
It should be irrelevant as long as we do not try to minimize compile time at all cost.